SIS_import/sis_set_url.rb

```diff
@@ -94,6 +77,25 @@ def getESBToken
 	end
 end
 
+
+# get a new esbToken
+def refreshESBToken()
+	encoded_string = Base64.strict_encode64(@esbKey + ":" + @esbSecret)
+	param_hash={"grant_type" => "client_credentials", "scope" => "PRODUCTION"}
+	responseBody = ESB_APICall(@esbTokenUrl + "/token?grant_type=client_credentials&scope=PRODUCTION",
+	                   "Basic " + encoded_string,
+	                   "application/x-www-form-urlencoded",
+	                   "POST",
+	                   param_hash)
+	json = json_parse_safe(@esbTokenUrl, responseBody, @logger)
+	if (!json.nil?)
+		@esbToken = json["access_token"]
+		@logger.info "ESB token refreshed at " + Time.now.to_s
```

Comment:
It would be be nice to print the first 4 or 5 characters of the token so it is clear what token is being used.